src/services/pending-queue.ts

```diff
@@ -192,17 +195,11 @@ export async function cancelEditingPendingQueueSubmission(
  * @returns
  */
 export function getPendingQueueSubmissions(): Promise<PendingFormSubmission[]> {
-  return utilsService.localForage
-    .getItem('submissions')
+  return utilsService
+    .getLocalForageItem(PENDING_QUEUE_SUBMISSIONS_KEY)
     .then((submissions) => (Array.isArray(submissions) ? submissions : []))
 }
 
-export function getFormSubmission(
```

Comment:
I think we'll have to keep doing this to handle existing pending queue items

Reply:
Pending submissions indexdb key has changed, there wont be any existing pending submissions.

Reply:
Fixing now.